[2024-06-06T22:45:26.627+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-06T22:45:26.671+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_pipeline_v2.validate_schema manual__2024-06-06T22:45:18.135998+00:00 [queued]>
[2024-06-06T22:45:26.690+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_pipeline_v2.validate_schema manual__2024-06-06T22:45:18.135998+00:00 [queued]>
[2024-06-06T22:45:26.692+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-06-06T22:45:26.730+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): validate_schema> on 2024-06-06 22:45:18.135998+00:00
[2024-06-06T22:45:26.746+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=193) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-06T22:45:26.750+0000] {standard_task_runner.py:63} INFO - Started process 195 to run task
[2024-06-06T22:45:26.749+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'data_pipeline_v2', 'validate_schema', 'manual__2024-06-06T22:45:18.135998+00:00', '--job-id', '61', '--raw', '--subdir', 'DAGS_FOLDER/***.py', '--cfg-path', '/tmp/tmpww2tnvn4']
[2024-06-06T22:45:26.754+0000] {standard_task_runner.py:91} INFO - Job 61: Subtask validate_schema
[2024-06-06T22:45:26.785+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2024-06-06T22:45:26.898+0000] {task_command.py:426} INFO - Running <TaskInstance: data_pipeline_v2.validate_schema manual__2024-06-06T22:45:18.135998+00:00 [running]> on host be153ff008c0
[2024-06-06T22:45:27.121+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='sanket' AIRFLOW_CTX_DAG_ID='data_pipeline_v2' AIRFLOW_CTX_TASK_ID='validate_schema' AIRFLOW_CTX_EXECUTION_DATE='2024-06-06T22:45:18.135998+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-06T22:45:18.135998+00:00'
[2024-06-06T22:45:27.125+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-06T22:45:27.127+0000] {python.py:237} INFO - Done. Returned value was: False
[2024-06-06T22:45:27.128+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-06T22:45:27.173+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=data_pipeline_v2, task_id=validate_schema, run_id=manual__2024-06-06T22:45:18.135998+00:00, execution_date=20240606T224518, start_date=20240606T224526, end_date=20240606T224527
[2024-06-06T22:45:27.212+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-06T22:45:27.246+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-06T22:45:27.252+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
